tests/test_core.py

```diff
@@ -20,7 +26,7 @@
     <subfield code="a">10.5281/zenodo.17575</subfield>
   </datafield>
   <datafield tag="520" ind1=" " ind2=" ">
-    <subfield code="a">&lt;p>Model definitions and data...&lt;/p></subfield>
+    <subfield code="a">&lt;p&gt;Model definitions and data...&lt;/p&gt;</subfield>
```

Comment:
I also needed to amend the test record. There was no clear way to avoid encoding the `>`.